rulesets/debug.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "./schema.json",
+  "version": 2,
+  "axioms": {
+    "linguist": "language",
+    "licensee": "license",
+    "packagers": "packager"
+  },
+  "rules": {
+    "philips-internal-com-migrationv2": {
+      "level": "warning",
+      "rule": {
+        "type": "file-not-contents",
+        "options": {
+          "globsAll": ["CNAME"],
+          "content": ".*\\.(philips-internal.com)",
+          "fail-on-non-existent": false,
+          "branches": ["default", "origin/gh-pages"]
```

Comment:
In this case, both the default branch (f.e. `main`) and `origin/gh-pages` are checked!

Reply:
Yes :) That is correct.